src/ontology/mondo-edit.obo

```diff
@@ -419922,10 +419925,13 @@ xref: SCTID:768555009 {source="MONDO:equivalentTo"}
 xref: UMLS:C4708510 {source="MONDO:equivalentTo", source="MONDO:MEDGEN", source="MEDGEN:1636705"}
 is_a: MONDO:0002254 {source="https://orcid.org/0000-0001-5208-3432"} ! syndromic disease
 is_a: MONDO:0016904 {source="Orphanet:314655"} ! partial deletion of the long arm of chromosome 5
+is_a: MONDO:1060108 {source="https://orcid.org/0000-0001-9310-0163"} ! PURA-related severe neonatal hypotonia-seizures-encephalopathy syndrome
```

Comment:
Is there a PMID or another source to support this SC? 
It is always best to have additional sources, in addition to "a single person".